aten/src/ATen/native/quantized/cpu/qconcat.cpp

```diff
@@ -0,0 +1,63 @@
+#include <ATen/ATen.h>
+#include <ATen/core/op_registration/op_registration.h>
+#include <ATen/native/TensorIterator.h>
+#include <ATen/native/cpu/Loops.h>
+
+#include <algorithm>
+#include <vector>
+
+namespace at { namespace native {
+namespace {
+
+bool is_valid_quantization_type(Tensor t) {
```

Comment:
we actually didn't do this check for other ops, I think we can assume the qschemes are valid.. (or this kind of check should actually be done in a base class for all quantized operators).

Reply:
oh looks like this is specific to quantized cat, then we can keep it.